### PR TITLE
add beta macos desktop blog post

### PR DESCRIPTION
**Pull request by bryanoltman**

### CODE REVIEW

File: src/content/blog/macos-beta.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This support is in beta and only supports Arm (Apple Silicon) macs at this time.
+We intend to support Intel macs in the future, but that may still be several
+weeks away.
+
+The release and patch flow is the same as you’re used to with iOS and Android:
+
+```sh
+# Creates a new macOS release
+shorebird release macos
```
Comment (eseidel) [inline src/content/blog/macos-beta.md:18]: Does `flutter` use macos or mac?

File: src/content/blog/macos-beta.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author: bryanoltman
+description: Announcing the beta release of macOS desktop support
+date: 2024-12-16
+---
+
+Shorebird now supports updating Flutter apps built for Mac desktop.
+
+This support is in beta and only supports Arm (Apple Silicon) macs at this time.
+We intend to support Intel macs in the future, but that may still be several
+weeks away.
```
Comment (eseidel) [inline src/content/blog/macos-beta.md:12]:
```suggestion
weeks away.  We're beginning Windows desktop support next and you can follow our progress [on GitHub](https://github.com/shorebirdtech/shorebird/issues/397).
```

**Review by eseidel — APPROVED** [review]

File: src/content/blog/macos-beta.md (hunk 1 above)
Comment (bryanoltman) [reply src/content/blog/macos-beta.md:18]: `macos`

Comment (eseidel) [reply src/content/blog/macos-beta.md:19]: 😮

**Pull request merged**